File: 20211204/react/src/models/Task.ts
export default class Task {
  public id: number;

  public title: string;

  constructor({ id, title }: {
    id?: number;
    title?: string;
  } = {}) {
    this.id = id || new Date().getTime();
    this.title = title || '';
  }

  updateTitle(title: string) {
    return new Task({ id: this.id, title });
  }
}

File: 20230207/java/app/src/main/java/com/ahastudio/todo/models/Task.java
package com.ahastudio.todo.models;

public class Task {
    private final int id;

    private final String title;

    public Task(int id, String title) {
        this.id = id;
        this.title = title;
    }

    public int id() {
        return id;
    }

    public String title() {
        return title;
    }
}
